Content.Client/Verbs/UI/VerbMenuPresenter.cs

```diff
@@ -170,7 +170,13 @@ public override void OnKeyBindDown(ContextMenuElement element, GUIBoundKeyEventA
                 return;
 
             if (element is not VerbMenuElement verbElement)
+            {
+                if (element is not ConfirmationMenuElement confElement)
+                    return;
+
+                ExecuteVerb(confElement.Verb, confElement.Type);
```

Comment:
IMO it would be a bit easier to make sense of the code if you move this outside of the if block and just make it
```cs
if (element is ConfirmationMenuElement confElement)
{
    ExecuteVerb(confElement.Verb, confElement.Type);
    return;
}
```

Reply:
I made it this way because the confirmation menu is rare with only three verbs using it at the moment.

Reply:
fair point.